darts/timeseries.py

```diff
@@ -806,6 +823,28 @@ def from_times_and_values(times: pd.DatetimeIndex,
             df.columns = columns
         return TimeSeries(df, freq, fill_missing_dates)
 
+    def create_dummy_index(self, start="19700101", freq="S") -> 'TimeSeries':
```

Comment:
Probably this method should be private (i.e. rename `_create_dummy_index`)

Reply:
Done